src/elements/content-picker/__tests__/cellRendererHelper.test.tsx

```diff
@@ -0,0 +1,47 @@
+import isRowSelectable from '../cellRendererHelper';
+import { BoxItem } from '../../../common/types/core';
+import { TYPE_FILE, TYPE_FOLDER } from '../../../constants';
+
+describe('elements/content-picker/cellRendererHelper', () => {
+    const createItem = (type: string, extension = '', selected = false): BoxItem => ({
+        id: '123',
+        name: 'test',
+        type,
+        extension,
+        selected,
+    });
+
+    describe('isRowSelectable()', () => {
```

Comment:
this function explicitly returns a boolean so the expects should probably `toBe(true)` / `toBe(false)` rather than `toBeTruthy()` or `toBeFalsy()`